ui/app/providers/configure/provider-form.tsx

```
@@ -584,6 +819,121 @@
 						{/* Container for Tab Content */}
 						<div className="relative">
 							<div>
+								{/* Custom Settings Tab */}
+								{isCustomProvider && selectedTab === "api-structure" && (
+									<div className="space-y-6">
+										<div className="space-y-4">
+											<div>
+												<label className="mb-2 block text-sm font-medium">Provider Name</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)